x-pack/plugins/security_solution/public/timelines/components/row_renderers_browser/examples/system_fim.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { mockEndgameFileCreateEvent } from '../../../../common/mock/mock_endgame_ecs_data';
+import { createFimRowRenderer } from '../../timeline/body/renderers/system/generic_row_renderer';
+import { ROW_RENDERER_BROWSER_EXAMPLE_TIMELINE_ID } from '../constants';
+
+const SystemFimExampleComponent: React.FC = () => {
+  const systemFimRowRenderer = createFimRowRenderer({
+    actionName: 'file_create_event',
+    text: 'created a file',
```

Comment:
consider using the `i18n` `CREATED_FILE` from `x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/system/translations.ts`